parent-pom.xml

```diff
@@ -19,7 +19,7 @@
     <apache.commons.text.version>1.10.0</apache.commons.text.version>
     <apache.httpclient.version>4.5.14</apache.httpclient.version>
     <apache.httpcore.version>4.4.16</apache.httpcore.version>
-    <arrow.version>10.0.1</arrow.version>
+    <arrow.version>15.0.0</arrow.version>
```

Comment:
Hmmm, why this version has changed? We have a very special version of Arrow forked from original repository with some fixes.

Reply:
I had to do this to get the test working locally. See the description. I saw that there is a PR to upgrade arrow so I will remove this and rebase when it is merged.